csrankings-k.csv

```diff
@@ -577,6 +577,7 @@ Kevin Waugh,Open University UK,http://stem.open.ac.uk/people/kgw9,l5ryKEkAAAAJ
 Kevin Y. Yip,Chinese University of Hong Kong,http://www.cse.cuhk.edu.hk/~kevinyip,pAD5GlEAAAAJ
 Kevyn Collins-Thompson,University of Michigan,http://www-personal.umich.edu/~kevynct,q__pgY9AkUQJ
 Kewei Tu,ShanghaiTech University,https://sist.shanghaitech.edu.cn/sist_en/2020/0814/c7582a54884/page.htm,5gi3Pm0AAAAJ
+Kexin Rong,Georgia Institute of Technlogy,https://kexinrong.github.io/,C8LDYK0AAAAJ

```

Comment:
There is a typo in the institution's name. It should be "Georgia Institute of **Technology**" instead of "Georgia Institute of **Technlogy**"